<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BrowserPanel</class>
 <widget class="QWidget" name="BrowserPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>2</number>
   </property>
   <item row="0" column="5">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>6</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="14">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>50</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QFrame" name="navigation_frame">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QSplitter" name="splitter_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <widget class="QTreeWidget" name="outline_tree">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="baseSize">
              <size>
               <width>0</width>
               <height>0</height>
              </size>
             </property>
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <property name="itemsExpandable">
              <bool>true</bool>
             </property>
             <property name="expandsOnDoubleClick">
              <bool>false</bool>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
            <widget class="QTreeWidget" name="browser_bookmark_view">
             <property name="rootIsDecorated">
              <bool>false</bool>
             </property>
             <column>
              <property name="text">
               <string>Favorites</string>
              </property>
             </column>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QFrame" name="frame_3">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="lay::BrowserTextWidget" name="browser">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="search_frame">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <property name="spacing">
              <number>2</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>  Find on page  </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="on_page_search_next">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../../lay/lay/layResources.qrc">
                 <normaloff>:/find.png</normaloff>:/find.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="on_page_search_edit"/>
             </item>
             <item>
              <widget class="QToolButton" name="search_close_button">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../../lay/lay/layResources.qrc">
                 <normaloff>:/clear_edit.png</normaloff>:/clear_edit.png</iconset>
               </property>
               <property name="autoRaise">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="forward_pb">
     <property name="toolTip">
      <string>Forward</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../lay/lay/layResources.qrc">
       <normaloff>:/forward.png</normaloff>:/forward.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QToolButton" name="back_pb">
     <property name="toolTip">
      <string>Back</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../lay/lay/layResources.qrc">
       <normaloff>:/back.png</normaloff>:/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QToolButton" name="home_pb">
     <property name="toolTip">
      <string>Home</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../lay/lay/layResources.qrc">
       <normaloff>:/home.png</normaloff>:/home.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>6</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="6">
    <widget class="QToolButton" name="prev_topic_pb">
     <property name="toolTip">
      <string>Previous Topic</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../lay/lay/layResources.qrc">
       <normaloff>:/prev_topic.png</normaloff>:/prev_topic.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="8">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>10</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="11">
    <widget class="QToolButton" name="search_button">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../lay/lay/layResources.qrc">
       <normaloff>:/find.png</normaloff>:/find.png</iconset>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="7">
    <widget class="QToolButton" name="next_topic_pb">
     <property name="toolTip">
      <string>Next Topic</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../lay/lay/layResources.qrc">
       <normaloff>:/next_topic.png</normaloff>:/next_topic.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="12">
    <widget class="QLineEdit" name="search_edit">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item row="0" column="10">
    <spacer>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="9">
    <widget class="QToolButton" name="bookmark_pb">
     <property name="toolTip">
      <string>Add this location to the favorites list</string>
     </property>
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../../lay/lay/layResources.qrc">
       <normaloff>:/bookmark.png</normaloff>:/bookmark.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="autoRaise">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
  <action name="action_find">
   <property name="text">
    <string>Find</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="action_bookmark">
   <property name="text">
    <string>Bookmark</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="action_delete_bookmark">
   <property name="text">
    <string>Delete Entry</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>lay::BrowserTextWidget</class>
   <extends>QTextBrowser</extends>
   <header>layBrowserPanel.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>search_edit</tabstop>
  <tabstop>back_pb</tabstop>
  <tabstop>forward_pb</tabstop>
  <tabstop>home_pb</tabstop>
 </tabstops>
 <resources>
  <include location="../../lay/lay/layResources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>search_close_button</sender>
   <signal>clicked()</signal>
   <receiver>search_frame</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>816</x>
     <y>571</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>577</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
